lib/foreman_theme_satellite/engine.rb

```diff
@@ -6,21 +6,11 @@
 module ForemanThemeSatellite
   class Engine < ::Rails::Engine
     engine_name 'foreman_theme_satellite'
-
-    config.eager_load_paths += Dir["#{config.root}/app/overrides"]
-    config.eager_load_paths += Dir["#{config.root}/app/controllers/concerns"]
-    config.eager_load_paths += Dir["#{config.root}/app/helpers"]
-    config.eager_load_paths += Dir["#{config.root}/app/models/concerns"]
-    config.eager_load_paths += Dir["#{config.root}/app/services/concerns"]
-    config.eager_load_paths += Dir["#{config.root}/lib/foreman_theme_satellite"]
-
-    initializer 'foreman_theme_satellite.load_default_settings', :before => :load_config_initializers do |app|
-      SettingRegistry.prepend SettingRegistryBranding
-    end
+    config.eager_load_paths += Dir["#{config.root}/lib"]
 
     initializer 'foreman_theme_satellite.register_plugin', :before=> :finisher_hook do |app|
       Foreman::Plugin.register :foreman_theme_satellite do
-        requires_foreman '>= 3.7.0'
+        requires_foreman '>= 3.13.0'
```

Comment:
We run CI with 3.9, so I assume it will fail now.

Reply:
It certainly does.